recipe/meta.yaml

```diff
@@ -1,33 +1,35 @@
-{% set version = "23.1.0" %}
-{% set hash = "090070a54572236f8e23263659520a27c31161e595a1c4feaffb70263d86a32e" %}
-{% set build_number = "0" %}
+{% set name = "conda-libmamba-solver" %}
+{% set version = "23.3.0" %}
 
 package:
-  name: conda-libmamba-solver
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://github.com/conda/conda-libmamba-solver/archive/refs/tags/{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://github.com/conda/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 0c209980d10d2acf3bb070f2a00c157319efc49f313e23992f7c350a37d98a87
+  folder: src/
 
 build:
-  skip: true  # [py<37]
-  # trigger 1
-  number: {{ build_number }}
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  number: 0
+  skip: true  # [py<38]
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{version}}
```

Comment:
https://conda-forge.org/docs/maintainer/knowledge_base.html#using-setuptools-scm is more details why this is needed